examples/src/components.cairo

```diff
@@ -1,6 +1,7 @@
 use array::ArrayTrait;
 
 #[derive(Component)]
+#[component(Indexed)]
```

Comment:
I think we want to use the named parameter syntax here, since in the future we can support other options, for example, data availability with something like `#[component(indexed = true, storage = "voltion")]`


```suggestion
#[component(indexed = true)]
```